File: pubsub/src/main/java/glitch/pubsub/events/MessageDeleteEvent.java
package glitch.pubsub.events;

import glitch.api.ws.events.IEvent;
import glitch.pubsub.GlitchPubSub;
import glitch.pubsub.Topic;
import glitch.pubsub.object.json.MessageDelete;

public class MessageDeleteEvent extends AbstractMessageEvent<MessageDelete> implements IEvent<GlitchPubSub> {
    public MessageDeleteEvent(GlitchPubSub client, Topic topic, MessageDelete data) {
        super(client, topic, data);
    }
}

File: pubsub/src/main/kotlin/glitch/pubsub/events/MessageDeleteEvent.kt
package glitch.pubsub.events

import glitch.api.ws.events.IEvent
import glitch.pubsub.GlitchPubSub
import glitch.pubsub.Topic
import glitch.pubsub.`object`.json.MessageDelete

class MessageDeleteEvent(client: GlitchPubSub, topic: Topic, data: MessageDelete) : AbstractMessageEvent<MessageDelete>(client, topic, data), IEvent<GlitchPubSub>
